packages/tools/webpack/extract-i18n-expressions-webpack-plugin.js

```diff
@@ -133,16 +126,36 @@ module.exports = class ExtractI18nExpressionsWebpackPlugin {
 
 		const translationCallExpressions = [];
 
-		this.translationsRegexps.forEach( ( regexp ) => {
-			[ ...source.matchAll( regexp ) ].forEach( ( [ callExpression ] ) => {
-				translationCallExpressions.push( callExpression );
+		[
+			...TRANSLATIONS_REGEXPS,
+			...COMMENTS_REGEXPS,
+		].forEach( ( regexp ) => {
```

Comment:
if you're now iterating also on comments, you should change the method name (`getTranslationCallExpressionsFromSubmodule()`)

Reply:
Done